hal/armv7m/imxrt/10xx/106x/imxrt.c

```diff
@@ -1659,7 +1660,7 @@ void _imxrt_init(void)
 
 	/* Disable unused clocks */
 	*(imxrt_common.ccm + ccm_ccgr0) = 0x00c0ffff;
-	*(imxrt_common.ccm + ccm_ccgr1) = 0x300c0000;
+	*(imxrt_common.ccm + ccm_ccgr1) = 0x300c0c00;
```

Comment:
no, this needs to be done in ethernet driver using platformctl on startup. If we're not running lwip, we don't want to have this clock enabled.